order/rpc_parse.go

```diff
@@ -62,6 +62,25 @@ func ParseRPCOrder(version, leaseDuration uint32,
 	}
 	kit.MinUnitsMatch = SupplyUnit(details.MinUnitsMatch)
 
+	switch details.ChannelType {
+	// Default value, trader didn't specify a channel type.
+	case auctioneerrpc.OrderChannelType_ORDER_CHANNEL_TYPE_UNKNOWN:
+		// TODO: Switch to script enforcement by default once we can
```

Comment:
Make sense, I wonder if there's something generic we can slot in _now_ to make this a bit more foolproof (harder for us to forget about it).

Reply:
We could look at lnd's feature flags on startup (similar as we do for finding out if a trader is tor only) and use the script enforced type for ask orders if the feature is turned on?

Reply:
@Roasbeef also had an idea of coming out with a new lease duration market where all orders signal script enforcement.